nuget.config

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
+    <add key="dotnet8" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" />
+  </packageSources>
+  <packageSourceMapping>
+    <!-- key value for <packageSource> should match key values from <packageSources> element -->
+    <packageSource key="nuget">
+      <package pattern="*" />
+    </packageSource>
+    <!-- <packageSource key="dotnet8">
+      <package pattern="FSharp.Core" />
+      <package pattern="FSharp.Compiler.Service" />
+    </packageSource> -->
```

Comment:
on nightly, we can just uncomment this mapping and update/outdated/etc commands will just work for these two packages. we can run `dotnet list package --outdated` and see what the updateable versions are